draft-irtf-cfrg-opaque.md

```diff
@@ -654,9 +656,19 @@ is that which is associated with the OPAQUE configuration (see {{configurations}
 See {{export-usage}} for details about the output export_key usage.
 
 Upon completion of this function, the client MUST send `record` to the server.
-The server then stores the tuple `(envU, pkU, kU, pkS, skS)`, where `envU` and `pkU`
-are obtained from `record`. If `skS` and `pkS` are used for multiple users, the
-server can store these values separately and omit them from the user's record.
+
+#### Credential File {#credential-file}
+
+The server then constructs and stores the `CredentialFile` object, where `envU` and `pkU`
+are obtained from `record`, and `kU` is retained from the output of `CreateRegistrationResponse`.
+
+~~~
+struct {
+    Scalar kU<1..2^16-1>;
```

Comment:
```suggestion
    SerializedScalar kU;
```

Reply:
I assume we want this to be serialized for storage, right?

Reply:
I could not find "SerializedScalar" in the VOPRF doc. Is there a definition for this somewhere?